src/plugins/intel_gpu/tests/unit/passes/prepare_buffer_fusing_test.cpp

```diff
@@ -363,3 +363,60 @@ TEST(prepare_buffer_fusing, crop_b_axis) {
         }
     }
 }
+
+TEST(prepare_buffer_fusing, crop_dynamic) {
+    auto& engine = get_test_engine();
+
+    auto input1 = layout{ ov::PartialShape{-1, 2, -1, -1}, data_types::f32, format::bfyx };
+    auto in_layout1 = layout{ ov::PartialShape{3, 2, 2, 2}, data_types::f32, format::bfyx };
+
+    topology topology;
+    topology.add(input_layout("Input", input1));
+    topology.add(crop("crop", input_info("Input"), tensor{1, 2, 2, 2}, tensor(1, 0, 0, 0)));
+    topology.add(reorder("reorder", input_info("crop"), format::bfyx, data_types::i8));
```

Comment:
BTW as I know the current situation to fix is when it is shape_of subgraph isnt it? Is this test for that purpose?

Reply:
It's removed.